### ISSUE TITLE
[Feature Request] No aggregation Windows

### ISSUE THREAD

**Issue by p62arrav** [GridProtectionAlliance/STTP2HIDS#1]
Hi, so I've been working with this tool for a while to send data from OpenPDC to an Influxdb 2.0 database. I really would like to have an option where data pass through this tool without any aggregation, so I can store PMU the same dataframes that are recorded. Right now, a Default Windows Size need to be defined.

**Comment by ritchiecarroll** [comment]
Are you saying you want to store multiple values for a given timestamp?

**Comment by p62arrav** [comment]
No, I just would like to store PMU data as they are measured, without any aggregation. Currently, you need to define a Windows size which agregate data for several milliseconds. I would like to have an option where this Aggregation Windows is 0.
